highfidelity/hifi
# Tablet ui audio screen

Test plan
- Press audio button on Tablet main menu
- make sure there is list of audio input and output devices
- make sure devices selectable
- make sure microphone muted and unmited
- make sure audio meter can be turned on and off
